src/components/inputs/multiline-text-input/multiline-text-input.js

```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 import requiredIf from 'react-required-if';
 import { useIntl } from 'react-intl';
 import { css } from '@emotion/core';
+import { AngleUpIcon, AngleDownIcon } from '@commercetools-uikit/icons';
```

Comment:
As far as I remember, one of the issues with tree shaking was related to the svgs...should we maybe publish each icon individually? If we bundle them all together we might end up with the same problem 🤔

Reply:
yeah, that's what I wanted to mess around with now. 👍 Going to investigate in the VRT app how treeshaking works with the icons.

Reply:
Ah yeah, so right now if you use `MultilineTextInput`, you end up with all of the icons. I'll look into alternatives now.